spyderlib/spyder.py

```diff
@@ -542,20 +542,24 @@ def setup(self):
                                         context=Qt.ApplicationShortcut)
             self.register_shortcut(self.close_dockwidget_action, "_",
                                    "Close pane")
-
+            self.lock_dockwidgets_action = create_action(self, _("Lock panes"),
+                                            toggled=self.toggle_lock_dockwidgets,
+                                            context=Qt.ApplicationShortcut)
+            self.register_shortcut(self.lock_dockwidgets_action, "_",
+                                       "Lock panes")

```

Comment:
@ccordoba12 would you prefer that instead of having a check in the menu action we changed it? so if panes are locked it should display `Unlock panes` and viceversa, if the panes are currently unlocked, it should display `Lock panes`?